src/uct/cuda/gdr_copy/gdr_copy_iface.c

```diff
@@ -104,21 +104,24 @@ static ucs_status_t
 uct_gdr_copy_estimate_perf(uct_iface_h iface, uct_perf_attr_t *perf_attr)
 {
     if (perf_attr->field_mask & UCT_PERF_ATTR_FIELD_BANDWIDTH) {
-        perf_attr->bandwidth.dedicated = 0;
         if (perf_attr->field_mask & UCT_PERF_ATTR_FIELD_OPERATION) {
             switch (perf_attr->operation) {
             case UCT_EP_OP_GET_SHORT:
             case UCT_EP_OP_GET_ZCOPY:
-                perf_attr->bandwidth.shared = 250.0 * UCS_MBYTE;
+                perf_attr->bandwidth.dedicated = 250.0 * UCS_MBYTE;
+                perf_attr->bandwidth.shared    = 0;
                 break;
             case UCT_EP_OP_PUT_SHORT:
-                perf_attr->bandwidth.shared = 10200.0 * UCS_MBYTE;
+                perf_attr->bandwidth.dedicated = 0;
+                perf_attr->bandwidth.shared    = 10200.0 * UCS_MBYTE;
                 break;
             default:
+                perf_attr->bandwidth.dedicated = 0;
```

Comment:
minor: align with the line below

Reply:
This alignment was suggested by CI's 'format code'.

Reply:
Reverted 1b9790d4d6e2e7044bdf231eb01ea14ab756aebf